backends/tfhe-cuda-backend/cuda/src/integer/multiplication.cuh

```diff
@@ -469,128 +449,6 @@ __host__ void scratch_cuda_integer_mult_radix_ciphertext_kb(
                                        allocate_gpu_memory);
 }
 
-// Function to apply lookup table,
-// It has two mode
-//  lsb_msb_mode == true - extracts lsb and msb
-//  lsb_msb_mode == false - extracts message and carry
-template <typename Torus, typename STorus, class params>
-void apply_lookup_table(Torus *input_ciphertexts, Torus *output_ciphertexts,
```

Comment:
Do we really want to remove this entry point? It seemed convenient :sweat_smile:

Reply:
yes it is old code